plugins/hr-resources/src/components/ScheduleView.svelte

```diff
@@ -95,18 +95,43 @@
       hr.class.Request,
       {
         'tzDueDate.year': { $gte: startDate.getFullYear() },
-        'tzDate.year': { $lte: endDate.getFullYear() }
+        'tzDate.year': { $lte: endDate.getFullYear() },
+        space: { $in: departments }
       },
       (res) => {
-        reqests = res
+        requests = res
+      }
+    )
+  }
+
+  const tempQ = createQuery()
+  let otherRequests: Request[] = []
+
+  function updateOther (staff: Staff[], startDate: Date, endDate: Date) {
+    console.log(staff)
+    tempQ.query(
+      hr.class.Request,
+      {
+        'tzDueDate.year': { $gte: startDate.getFullYear() },
+        'tzDate.year': { $lte: endDate.getFullYear() },
+        attachedTo: { $in: staff.map((p) => p._id) }
```

Comment:
you could just add _id: {$nin: requests.map((p) => p._id)} and concat with requests.